syntax = "proto3";

package greet;

// The greeting service definition.
service Greeter {
    // Sends a greeting
    rpc Greet(GreetRequest) returns (GreetResponse);
}

// The request message containing the user's name.
message GreetRequest {
    string name = 1;
}

// The response message containing the greeting message.
message GreetResponse {
    string message = 1;
}
